docs/Lab-Learning-Programs/README.MD

```diff
@@ -23,15 +23,14 @@ table th:nth-of-type(4) {
 
 Our team creates and offers learning programs to support agencies to build the structures and competencies that enable better service delivery through digital government. 
 
-The methods and tools used for delivery at the lab are largely new to to the BC Public Service, but have been used with success in other governments and sectors. We apply a specific approach, called Agile, to foster high performing teams that focus and deliver products in short time frames through iterative development. We also expect continuous service improvement and new approaches to technology, specifically DevOps practices.
+The methods and tools used for delivery at the lab are largely new to to the BC Public Service, but have been used with success in other governments and sectors. We apply a specific approach, called Agile, to foster high performing teams that focus and deliver products in short time frames through iterative development. We also expect continuous service improvement and new approaches to technology including DevOps practices.
 
 The language and culture that has developed around these approaches is new and can be disorienting if you have not made the effort to learn.
+* [Check out some of our favourite online resources.](https://bcgov.github.io/ExchangeLabOps/Lab-Learning-Programs/Learning-Links.html)
+* [Learn about the application of Agile in the BC Public Service.](
```

Comment:
is there supposed to be a link after "Learn about the application of... "?

Reply:
yup! nice catch.